hr_employee_age/tests/test_hr_employee.py

```diff
@@ -0,0 +1,15 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.tests import common
+
+
+class TestHrEmployee(common.TransactionCase):
+
+    def setUp(self):
+        super(TestHrEmployee, self).setUp()
+        self.emp_root = self.env.ref('hr.employee_root')
+        self.emp_root.write({
+            'birthday': '1990-05-15'
+        })
+
+    def test_compute_age(self):
+        self.emp_root._compute_age()
```

Comment:
This test only checks that the code runs correctly. It doesn't do any check about it providing the correct result.
Please, add some simple assertion on `self.emp_root.age` field.

Reply:
@espo-tony here an only call _compute_age method, and one more thing we pass `birthday` field. 

Ex. Today age is 28 but next year 29 that time assertion is False so we directly use this method
here we use manually `assertEqual(self.emp_root.age, 28)` but it's wrong to logically.

CC  @JayVora-SerpentCS

Reply:
@nikul-serpentcs Of course the assert needs to be dinamic: 28 shouldn't be hardcoded but computed within the test.

Reply:
@espo-tony compute nothing to return, we have only one variable like `self.emp_root.age`  then how to assert. and how to check another one.if you have any idea about that.

Reply:
Something like:

```
age = relativedelta(
    fields.Date.from_string(fields.Date.today()),
    fields.Date.from_string(self.emp_root.birthday)).years
assertEqual(self.emp_root.age, age)
```
Otherwise you can do something like this:

```
self.emp_root.birthday = fields.Date.from_string(
    fields.Date.today() - relativedelta(years=28))
self.emp_root._compute_age()
assertEqual(self.emp_root.age, 28) 
```
Or I guess you can find another couple of ways to achieve the same goal. The point is that the test as it's written is not proving the code is functionally correct because it doesn't do any assertion on its result.

Reply:
@espo-tony Ok, In UT one more time Calculate age and after check assertion.

Reply:
On the assert equal we are comparing two variables that are computed using the same instruction so the will obviously be equal. I know it is a simple function and probably works fine, but I think it would be better to use the second option proposed by @espo-tony or even using mock to control what fields.Date.today() returns and compare the age against what we expect.

Reply:
@jarroyomorales nope, not ready ^